AutomatedTesting/Gem/PythonTests/Atom/tests/MaterialCanvas_Atom_BasicTests.py

```diff
@@ -0,0 +1,135 @@
+    material_graph_name_is_test1 = (
+        "Verified material graph name is 'test1'",
+        "P0: Failed to verify material graph name is 'test1'")
+
+
+def MaterialCanvas_BasicFunctionalityChecks_AllChecksPass():
+    """
+    Summary:
+    Tests basic MaterialCanvas functionality but does not deal with file I/O (i.e. saving).
+
+    Expected Behavior:
+    All MaterialCanvas basic tests pass.
+
+    Test Steps:
+    1) Open an existing material graph document.
+    2) Close the selected material graph document.
+    3) Open multiple material graph documents then use the CloseAllDocuments bus call to close them all.
+    4) Open multiple material graph documents then verify all material documents are opened.
+    5) Use the CloseAllDocumentsExcept bus call to close all but one.
+    6) Verify Node Palette pane visibility.
+    7) Verify material graph name is 'test1'.
+    8) Look for errors and asserts.
+
+    :return: None
+    """
+
+    import os
+
+    import Atom.atom_utils.atom_tools_utils as atom_tools_utils
+    import Atom.atom_utils.material_canvas_utils as material_canvas_utils
+    from editor_python_test_tools.utils import Report, Tracer, TestHelper
+
+    with Tracer() as error_tracer:
+        # Set constants before starting test steps.
+        test_1_material_graph = os.path.join(atom_tools_utils.MATERIALCANVAS_GRAPH_PATH, "test1.materialgraph")
+        test_2_material_graph = os.path.join(atom_tools_utils.MATERIALCANVAS_GRAPH_PATH, "test2.materialgraph")
+        test_3_material_graph = os.path.join(atom_tools_utils.MATERIALCANVAS_GRAPH_PATH, "test3.materialgraph")
+        test_4_material_graph = os.path.join(atom_tools_utils.MATERIALCANVAS_GRAPH_PATH, "test4.materialgraph")
+        test_5_material_graph = os.path.join(atom_tools_utils.MATERIALCANVAS_GRAPH_PATH, "test5.materialgraph")
+
+        # 1. Open an existing material graph document.
+        material_document_id = atom_tools_utils.open_document(test_1_material_graph)
+        Report.result(
+            Tests.open_existing_material_graph,
+            atom_tools_utils.is_document_open(material_document_id) is True)
+
+        # 2. Close the selected material graph document.
+        atom_tools_utils.close_document(material_document_id)
+        Report.result(
+            Tests.close_opened_material_graph,
+            atom_tools_utils.is_document_open(material_document_id) is False)
+
+        # 3. Open multiple material graph documents then use the CloseAllDocuments bus call to close them all.
+        for material_graph_document in [test_1_material_graph, test_2_material_graph, test_3_material_graph,
+                                        test_4_material_graph, test_5_material_graph]:
+            atom_tools_utils.open_document(os.path.join(material_graph_document))
+        Report.result(
+            Tests.close_all_opened_material_graphs,
+            atom_tools_utils.close_all_documents() is True)
+
```

Comment:
Do you need to add section here to verify that all material graph documents are closed?

Looks like you verify this below with the close all but one test. 
Example: "atom_tools_utils.is_document_open(material_graph_2_document_id) is False"

Reply:
`atom_tools.utils.close_all_documents()` returns a bool and there are 2 close tests: `Tests.close_all_opened_material_graphs` + `Tests.close_all_opened_material_graphs_except_one`. So closing all is already covered in addition to closing all EXCEPT one.

Reply:
ah gotcha, makes sense. I'll approve then